### DISCUSSION TITLE
Need help with some questions

### DISCUSSION THREAD

**Discussion by elareh** [flutter-stripe/flutter_stripe#1265] [Q&A]
Hi All! We are doing an flutter stripe integration in our mobile App and we had some questions:

- Do you have any examples of the UI integrated in the flutter_stripe package?

- If you do, do you have a widget to display a card with some details / a list of cards, or does it need to be developed on our own?

-If there are no prebuilt widgets, what endpoints can we query and with what credentials to retrieve the data of cards registered by a user and allow them to edit or delete them?

-How is the payment of taxes to the app stores handled for each transaction within the app?

-Show in-depth the feature of splitting payment percentages.

-Customization of forms for registering new cards.

-Explanation of why you require the use of FlutterFragmentActivity and CompatTheme.

-Regarding the integration with Google Pay / Apple Pay:
        -Do you have a different interface from the native one for each store? Or do you follow the one from the flutter_pay package?
        -When transacting through this feature, can we access the benefit of splitting payment percentages to the intended recipients?

Thank you

**Comment by remonh87** [comment]
Hi, I would suggest to checkout our [example app](https://github.com/flutter-stripe/flutter_stripe/tree/main/example) . As widget use the [payment sheet](https://docs.page/flutter-stripe/flutter_stripe/sheet) as it provides the easiest and most user friendly way of handling payments. I wouldn't use the pay plugin myself as I consider it to be quite complicated to configure compared to the standard integration but I leave that up to you. The payment sheet has build in Apple Pay and google pay capabilities.
